test/e2e/page-objects/pages/account-list-page.ts

```diff
@@ -234,6 +247,72 @@ class AccountListPage {
     }
   }
 
+  async addAccount(accountType: ACCOUNT_TYPE, accountName: string) {
```

Comment:
What about using an optional here, so you don't have to pass `''` in every call for `accountName`?

```suggestion
  async addAccount(accountType: ACCOUNT_TYPE, accountName?: string) {
```